### ISSUE TITLE
Certain faces cannot be inspected

### ISSUE THREAD

**Issue by MatthiasJ1** [bernhard-42/vscode-ocp-cad-viewer#115]
In this example, the cylindrical faces cannot be inspected.

```py
t1 = SlotOverall(10, 5)
t2 = Pos(Z=2) * SlotOverall(8, 3)
t = loft([t1, t2])
t = offset(t, 1)
```

<img width="439" alt="Screenshot 1" src="https://github.com/user-attachments/assets/6be518ce-4f0e-43ee-af03-5bdf08e53b1c">
<img width="439" alt="Screenshot 2" src="https://github.com/user-attachments/assets/a632c4fe-dc52-4154-be55-7b1fb5c6528a">

**Comment by bernhard-42** [comment]
Note to myself:
According to the backend Terminal output:
```
response.radius = shape._geom_adaptor().Cylinder().Radius()
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'OCP.Geom.Geom_RectangularTrimmedSurface' object has no attribute 'Cylinder'
```

check out which results `shape._geom_adaptor()` could have and handle them

**Comment by bernhard-42** [comment]
https://dev.opencascade.org/doc/refman/html/classGeom__Surface.html
![image](https://github.com/user-attachments/assets/812ec562-6927-44e5-9b69-898f94ee8708)

**Comment by bernhard-42** [comment]
Fixed in 2.6.1
